_rules/aria-required-context-role-ff89c9.md

```diff
@@ -17,214 +17,239 @@ input_aspects:
 
 ```html
-<div role="list" aria-owns="item"></div>
-<div id="item" role="listitem">List item 1</div>
+<div role="list" aria-owns="item1 item2"></div>
+<div id="item1" role="listitem">List item 1</div>
+<div id="item2" role="listitem">List item 2</div>
 ```
 
 #### Passed Example 5
 
-The `aria-owns` attribute override normal DOM tree relationship. Thus, the innermost `div` element (`item`) is not owned by the intermediate one (with a `role="navigation"`) and has the correct [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+These elements with an [explicit role][] of `listitem` are [owned by][] an element with their [required context role][] even though they are not direct DOM descendants of it. The `aria-owns` attribute is used to alter the accessibility tree and place the target elements in their [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
 
 ```html
-<div role="list" aria-owns="item">
+<div role="list" aria-owns="item1 item2">
 	<div role="navigation">
-		<div id="item" role="listitem">List item 1</div>
+		<div id="item1" role="listitem">List item 1</div>
+		<div id="item2" role="listitem">List item 2</div>
 	</div>
 </div>
 ```
 
 #### Passed Example 6
 
-Since implicit ownership can cross shadow boundaries, the element with the [explicit semantic role](#explicit-role) of `listitem` is contained within its [required context role](https://www.w3.org/TR/wai-aria-1.1/#scope) of `list`.
+These elements with an [explicit role][] of `listitem` are [owned by][] an element with their [required context role][] because implicit [ownership][owned by] (inherited from DOM tree structure) crosses shadow boundaries.
 
 ```html
 <div id="host" role="list"></div>
 
 <script>
 	const host = document.querySelector('#host')
 	const root = host.attachShadow({ mode: 'open' })
-	root.innerHTML = '<div role="listitem">List item 1</div>'
+	root.innerHTML = '<div role="listitem">List item 1</div> <div role="listitem">List item 2</div>'
 </script>
 ```
 
 ### Failed
 
 #### Failed Example 1
 
-The `listitem` has no [context role](https://www.w3.org/TR/wai-aria-1.1/#scope).
+This element with an [explicit role][] of `listitem` is not [owned by][] an element with its [required context role][].
 
 ```html
 <div role="listitem">List item 1</div>
 ```
 
 #### Failed Example 2
 
-The `listitem` is owned by the `tabpanel`, because it is the closest ancestor, but `tabpanel` is not the correct [context](https://www.w3.org/TR/wai-aria-1.1/#scope) for `listitem`.
+These elements with an [explicit role][] of `listitem` are not [owned by][] an element with their [required context role][]. They are [owned by][] the `tabpanel` element, because it is the closest ancestor.
```

Comment:
Is there a need to point out that 'tabpanel' applies because it is in the DOM/accessibility tree, where previously 'presentation' was not?

Reply:
I think this is OK because this is the intuitive way (the accessibility tree is essentially the DOM tree).
Whereas Passed Example 3 is relying on the differences between accessibility tree and DOM and thus somewhat counter-intuitive and requires the explanation.